doc/040_backup.rst

```diff
@@ -407,6 +407,7 @@ environment variables. The following lists these environment variables:
 
 .. code-block:: console
 
+    RESTIC_REPOSITORY_FILE              Name of a file containing the location of the repository (replaces --repository-file)
```

Comment:
I'd suggest formulating this the same way as below for `RESTIC_PASSWORD_FILE`. If the reader things it's unclear what it does, they're expected to read the help for `--repository-file`.

Reply:
If you don't like the former I'd probably shorten it to "Name of file containing the repository location (replaces --repository-file)".